src/frontends/tensorflow/src/op/concat.cpp

```diff
@@ -20,20 +20,21 @@ OutputVector translate_concat_op(const NodeContext& node) {
     // axis is the first input for Concat
     // and is the last input to ConcatV2
     default_op_checks(node, 2, {"Concat", "ConcatV2"});
-    auto input_size = node.get_input_size();
+    auto input_size = static_cast<int>(node.get_input_size());
```

Comment:
why was it changed?

Reply:
It is changed because `input_size` is used below in `for` loop to align with type of `input_idx`.

Reply:
I see `node.get_input` method accepts `int` type.
ok